packages/graphql-transformer-common/src/ResolverResourceIDs.ts

```diff
@@ -1,6 +1,9 @@
 import { graphqlName, toUpper } from "./util";
 
 export class ResolverResourceIDs {
+    static DynamoDBTableResourceID(typeName: string): string {
```

Comment:
Duplicate code, we already have a function for this: ```ModelResourceIDs.ModelTableResourceID```, please use that.

Reply:
changed in the latest commit